core/src/test/java/io/questdb/test/griffin/wal/WalAlterTableSqlTest.java

```diff
@@ -158,6 +158,30 @@ public void createWalAndDropPartition() throws Exception {
         });
     }
 
+
```

Comment:
Would it make sense to also test dropping non-existing partition past the active one ?

Reply:
@bziobrowski added the drops, please have a look